submissions/Serge044/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Serge's popup project</title>
+    <link rel="shortcut icon" href="images/favicon.png">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav__menu">
+                <ul class="ul__items">
+                    <li>
+                        <a href="#Mail" class="main__nav-buttons mail"><span
```

Comment:
Please, read more about [BEM](https://en.bem.info/methodology/)
I don't see class name of block `main__nav`